### DISCUSSION TITLE
FOUNDATIONS: Implement & Integrate Command Tools

### DISCUSSION THREAD

**Discussion by mabroukmahdhi** [cjdutoit/Standardly#70] [Ideas]
I am thinking about Package Manager Console Commands, so something like :
PM> Add-Broker Student
PM> Add-Service Student 
PM> Test-Service StudentService 
PM> ....

**Comment by coeur85** [comment]
will that work from cli outside vs ?

**Comment by mabroukmahdhi** [comment]
> will that work from cli outside vs ?

It should...

**Comment by cjdutoit** [comment]
@mabroukmahdhi that would be a nice addition.    I think at a minimum you would need to pass in template, singular and plural names with the CLI command.   The rest we can infer from the config options if not provided.

> dotnet standardly templates        -> list all available templates
> dotnet standardly add %templatename% %singular name% %plural name%
> dotnet standardly add %templatename% %singular name% %plural name% --projectname %projectname% --username %username%

**Reply by Spectralgo** [reply]
Here is a resource to help us build an interactive command line tool:
https://youtu.be/MOweq3EttPU
